pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1353,6 +1404,10 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
         }
 
         await this.progressTracker.action('Running provisioning scripts', 100, this.runProvisioningScripts());
+
+        if (config.experimental.virtualMachine.proxy.enabled && config.experimental.virtualMachine.proxy.address && config.experimental.virtualMachine.proxy.port) {
```

Comment:
Does this need another condition to wrap this condition? Since your work relies on the new networking stack we may need to wrap this or include the following condition:

```
if (this.cfg?.experimental.virtualMachine.networkingTunnel)...
```

Reply:
As discussed, it's working both with and without the "networkingTunnel".